superduperdb/container/vector_index.py

```diff
@@ -99,7 +100,10 @@ def on_load(self, db: DB) -> None:
                 create=True,  # type: ignore[call-arg]
             )
 
-        if not s.CFG.cluster.cdc:
+        assert hasattr(self.indexing_listener.select, 'collection')
+        clt = self.indexing_listener.select.collection
```

Comment:
Ok for now, but we need a concept which isn't restricted to MongoDB

Reply:
Probably we should share some/ many concepts between `ibis` and `pymongo` queries. E.g. we could have a `@property` called "collection_or_table_name", or something like that.